spec/support/E2E_README.md

```diff
@@ -21,42 +21,48 @@ Since E2E tests are expensive -- slow to run and fiddly to write/update -- we sh
     ```bash
     docker-compose run --rm shell  # `dcr shell` if you have that alias
     ```
-   
-3. In the Docker shell, run the `run_hmis_system_tests` script, indicating the branch of the `hmis-frontend` repo you want to test:
-    ```bash
-    BRANCH_NAME=branch-name-to-test bin/run_hmis_system_tests.sh
-    ```
-
-### Local development
 
-This script is designed to be run as part of a CI pipeline. It clones the frontend repo into a temporary directory, serves the frontend using yarn preview, runs the system tests against that frontend, and then shuts everything down and cleans up after the tests finish running. But when working on E2E tests locally, we don't want to repeatedly clone and clean up the whole repo on each run. Instead,
+3. In the Docker shell, run the `run_hmis_system_tests` script, indicating the branch of the `hmis-frontend` repo you want to test. This script will start the vite "preview" as a background process. The script will exit when complete, but the process will remaing running. You can use `ps aux` to find the PID.
 
-4. Open a new terminal tab and connect to the same Docker container from step 2. For example,
     ```bash
-    docker ps # grab the container ID for the Docker shell
-    docker exec -it $CONTAINER_ID /bin/bash # with the container ID from above
+    BRANCH_NAME=release-X ./bin/run_hmis_system_tests.sh --dev
     ```
-   You should be connected to the same Docker container in two tabs: one to serve the frontend, and the other to run the tests.
-   
-5. In the first tab, clone the repo and serve the frontend. You can do this by running the `bin/run_hmis_system_tests.sh` with the following modifications:
-    - comment out the `trap cleanup EXIT` line (we don't want to clean up after exiting the script)
-    - comment out everything after the `yarn preview` line
 
-6. In the other tab, run the `rspec` tests. Choosing just what we need from `bin/run_hmis_system_tests.sh`,
+4. Once that process has completed, you can kick off your `rspec` test(s).
     ```bash
-    HOSTNAME=`hostname`
-    unset HMIS_OKTA_CLIENT_ID
-    unset OKTA_DOMAIN
```

Comment:
This seems unnecessary to call out, since our default test env already does it: https://github.com/greenriver/hmis-warehouse/blob/8e0ec4f64eba4c9287f8e51e84a61fdc08924eb8/.env.test#L39-L40
@ttoomey maybe you have a local override file that was setting them, or something?

Reply:
Good spot, yes I think these env vars can be removed, it's probably not needed since we unset them in env.test. I don't remember why we had these originally